lighthouse-core/scripts/chromium-web-tests/run-web-tests.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+##
+# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LH_ROOT="$SCRIPT_DIR/../../.."
+
+if [ x"$BLINK_TOOLS_PATH" == x ]; then
+  echo "Error: Environment variable BLINK_TOOLS_PATH not set"
+  exit 1
+fi
+
+if [ x"$DEVTOOLS_PATH" == x ]; then
+  echo "Error: Environment variable DEVTOOLS_PATH not set"
+  exit 1
+fi
+
+unset -v latest_content_shell
+for file in "$LH_ROOT/.tmp/chromium-web-tests/content-shells"/*/; do
+  [[ $file -nt $latest_content_shell ]] && latest_content_shell=$file
+done
+
+# Roll devtools. Besides giving DevTools the latest lighthouse source files,
+# this also copies over the webtests.
+cd "$LH_ROOT"
+yarn devtools "$DEVTOOLS_PATH"
+cd -
```

Comment:
can we make this directory explicit? it's not super clear what directory this is supposed to be and we do it a few times in the script